test/Numerics/Mesh/topology.jl

```diff
@@ -4,69 +4,209 @@ using Combinatorics, MPI
 
 MPI.Initialized() || MPI.Init()
 
-@testset "cubedshellwarp tests" begin
-    import ClimateMachine.Mesh.Topologies: cubedshellwarp
+@testset "equiangular_cubed_shell_warp tests" begin
+    import ClimateMachine.Mesh.Topologies: equiangular_cubed_shell_warp
 
     @testset "check radius" begin
-        @test hypot(cubedshellwarp(3.0, -2.2, 1.3)...) ≈ 3.0 rtol = eps()
-        @test hypot(cubedshellwarp(-3.0, -2.2, 1.3)...) ≈ 3.0 rtol = eps()
-        @test hypot(cubedshellwarp(1.1, -2.2, 3.0)...) ≈ 3.0 rtol = eps()
-        @test hypot(cubedshellwarp(1.1, -2.2, -3.0)...) ≈ 3.0 rtol = eps()
-        @test hypot(cubedshellwarp(1.1, 3.0, 0.0)...) ≈ 3.0 rtol = eps()
-        @test hypot(cubedshellwarp(1.1, -3.0, 0.0)...) ≈ 3.0 rtol = eps()
+        @test hypot(equiangular_cubed_shell_warp(3.0, -2.2, 1.3)...) ≈ 3.0 rtol =
```

Comment:
Just a thought, it might be nice to make a short alias for `equiangular_cubed_shell_warp` so that we don't get hammered by the formatter

Reply:
Thanks for the idea. What alias would you have in mind in this case? I specifically wanted to use snake case for accessibility and readability though

Reply:
🤷🏻‍♂️ `warp_ecs`? Again, just a thought. Up to you.